frontend/public/components/utils/kebab.tsx

```diff
@@ -543,6 +544,7 @@ export type KebabAction = (
   obj: K8sResourceKind,
   resources?: any,
   customData?: any,
+  t?: TFunction,
```

Comment:
🤔  Has the Admin team not done any kebab translations? Seems odd for us to be adding it in this PR.

Reply:
No, I did not see any action translations yet.

Reply:
@vikram-raj Please check with the tmp-i18n channel. This feels weird, not sure what they are doing about this right now.

Reply:
You can use the `labelKey` property for now.